e2e/testcases/legacy_test.go

```diff
@@ -211,10 +208,11 @@ func TestBats(t *testing.T) {
 			),
 			skipAutopilotCluster: testNums(10), // ignore the test that mutates the kube-system namespace
 		},
-		{
-			fileName:      "cli.bats",
-			skipMultiRepo: allTests, // TODO: implement nomos status in CLI, this may be a go rewrite
-		},
+		// Fully converted into cli_test.go
+		//{
+		//	fileName:      "cli.bats",
+		//	skipMultiRepo: allTests, // TODO: implement nomos status in CLI, this may be a go rewrite
```

Comment:
`// TODO: implement nomos status in CLI, this may be a go rewrite`: this should be removed.